app/grandchallenge/algorithms/templates/algorithms/algorithm_detail.html

```diff
@@ -67,6 +70,15 @@ <h2>Jobs for this algorithm</h2>
                     </span>
                     </td>
                     <td>{{ job.result.output }}</td>
+                    {% if job.result.images.all %}
+                        <td>
+                            {% for image in job.result.images.all %}
+                                <a href="{{ job.image.cirrus_link }}&{{ settings.CIRRUS_ANNOATION_QUERY_PARAM }}={{ image.pk }}">
```

Comment:
Just to check - this works ok? You don't need to use `&amp;`?

Reply:
It works in Chrome but `&amp;` might indeed be safer/better so I changed it. 

Also, it actually was not outputting the correct URL because the settings object is not exposed in the template so I added `CIRRUS_ANNOTATION_QUERY_PARAM` to the context. Now it should work, but I can't test it in my dev environment because CIRRUS tries to get the images from the production environment.